web/js/Chart.js

```diff
@@ -38,34 +39,6 @@ function formatMilliseconds(milliseconds) {
   return formatSeconds(milliseconds / 1000);
 }
 
-/**
- * Formats hours into a human readable string
- * @param {number} hours - The time duration in hours
- * @returns {string} Formatted time string (e.g., "30m", "2h 30m", "1d 5h")
- */
-function formatHours(hours) {
```

Comment:
I committed this by mistake to `main` 🤦🏼. I refrained from reverting, and instead created this PR adding tests.